packages/web/src/components/pages/WebsiteResults.tsx

```diff
@@ -56,13 +58,16 @@ export function WebsiteResultsPage() {
 
   const availableFilters: PackageFilters = useMemo(
     () => ({
+      searchText: '',
```

Comment:
We could possible mark this as an optional field in the type itself and avoid specifying it here.